### ISSUE TITLE
Indexes not dropped

### ISSUE THREAD

**Issue by geekdenz** [silverstripe/silverstripe-postgresql#73]
We encountered an issue with the indexes not being dropped before they were created automatically in version 1.1.0.

I have a fix for it, but decided to log the issue to create a more transparent pull request.

**Comment by dhensby** [comment]
@geekdenz did you ever PR for this?

**Comment by geekdenz** [comment]
> @geekdenz did you ever PR for this?

No. Not yet anyway. Considering doing this now. If there's demand, I'll look into it. Please like or respond so I can prioritize.
